docs/source/examples/workspace/topologies/os-server-new.yml

```diff
@@ -16,7 +16,7 @@ resource_groups:
       - name: "{{ distro | default('') }}frontend"
         role: os_server
         flavor: m1.small
-        image: Fedora-Cloud-Base-26-compose-latest
+        image: CentOS-7-x86_64-GenericCloud-1612
```

Comment:
Does this need to be switched back?  I know sk had to change this because the CentOS image was missing? Or do I have it backwards?